test/content-type.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+const EXTENSION_TESTS = [
+  ['https://resources.whatwg.org/standard.css', 'text/css'],
+  ['https://whatwg.org/specs/web-workers/current-work/rationale.html', 'text/html; charset=utf-8'],
+  ['https://html.spec.whatwg.org/images/abstract.jpeg', 'image/jpeg'],
+  ['https://resources.whatwg.org/browser-logos/bb.jpg', 'image/jpeg'],
+  ['https://resources.whatwg.org/dfn.js', 'application/javascript'],
+  ['https://resources.whatwg.org/biblio.json', 'application/json'],
+  ['https://resources.whatwg.org/README.md', 'text/plain'],
+  ['https://resources.whatwg.org/logo.png', 'image/png'],
+  ['https://resources.whatwg.org/build/deploy.sh', 'text/plain'],
+  ['https://resources.whatwg.org/logo.svg', 'image/svg+xml'],
+  ['https://whatwg.org/robots.txt', 'text/plain; charset=utf-8'],
+  ['https://resources.whatwg.org/fonts/SourceSansPro-Regular.woff', 'application/font-woff'],
+  ['https://resources.whatwg.org/fonts/SourceSansPro-Regular.woff2', 'font/woff2'],
+  ['https://whatwg.org/demos/repeat-02/results.xml', 'application/xml'],
+];
+
+const WITHOUT_EXTENSION_TESTS = [
```

Comment:
Is this meant to be comprehensive? Seems a pain to keep this in sync every time we add a file....

Reply:
It's all extensionless files I could find currently, but no, I don't think we should keep updating this, I think we should stop adding such files, or at least only use it for HTML for pretty URLs.